appendixes/SS-behat-quick-setup.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# bash script for SS behat extension setup
+
+# set output color
+red=`tput setaf 1`
+green=`tput setaf 2`
+yellow=`tput setaf 3`
+reset=`tput sgr0`
+
+# get current project directory
+path="$( pwd )"
+
+# config "base_url" in behat.yml
+echo "${green}Copying behat.yml file...${reset}"
+cp -fv $path/vendor/silverstripe/behat-extension/appendixes/behat.yml $path/behat.yml
+echo "${green}Please enter the site URL which you want Behat test to run against and then press ENTER: "
+read base_url
+sed -i "" "s@base_url:.*@base_url: $base_url@g" $path/behat.yml
+echo "base_url: $base_url is set in your behat.yml file successfully!"
+
+# copy files for SS Behat test session running
+echo "\n${green}Copying files for SS Behat test session running...${reset}"
+
+cp -fv $path/vendor/silverstripe/behat-extension/appendixes/mysite/_config/behat.yml $path/mysite/_config/behat.yml
+echo "Appending TestSessionEnvironment and TestSessionController to $path/mysite/_config/config.yml"
+echo "\n" >> $path/mysite/_config/config.yml
+cat $path/vendor/silverstripe/behat-extension/appendixes/mysite/_config/config.yml >> $path/mysite/_config/config.yml
+cp -Rv $path/vendor/silverstripe/behat-extension/appendixes/mysite/code/testing/ $path/mysite/code/testing/
+cp -Rv $path/vendor/chillu/fakedatabase/src/ $path/mysite/code/testing/
```

Comment:
It shouldn't be necessary to copy `fakedatabase` files into `mysite`, right? Composer/PHP autoloading should make those files available from their `vendor/` location. If you need customisation of these, subclassing is your friend.